### PR TITLE
Story3 track layout

### PR DESCRIPTION
**Pull request by nsantiagoblair**
-> Track 1 layout
-> Track 2 layout
-> Navigation to and from tracks

🍋

### CODE REVIEW

**Comment by charliecog** [comment]
IE9 not working at all

File: css/splash.css [hunk 1]
```diff
@@ -39,13 +43,14 @@
...
     width: 350px;
-    margin: 0 auto;
+    margin: 50px auto 20px auto;
     border: 1px solid #000000;
     border-radius: 5px;
 }
 
 .splash .gameLevel h2  {
-    margin: 20px auto;
+    width: 350px;
```
Comment (charliecog) [inline css/splash.css]: unused

File: includes/trackLayout1.html [hunk 2]
```diff
@@ -0,0 +1,22 @@
+
```
Comment (charliecog) [inline includes/trackLayout1.html]: whitespace

File: index.html [hunk 3]
```diff
@@ -4,27 +4,29 @@
...
     <link rel="stylesheet" type="text/css" href="css/styles.css">
     <link rel="stylesheet" type="text/css" href="css/splash.css">
+    <link rel="stylesheet" type="text/css" href="css/track.css">
     <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
 </head>
 <body>
-    <main class="container">
-        <div class="splash">
-            <img src="assets/gameTitle.png" alt="Mayrio Kart">
+    <main id="container" class="container">
```
Comment (charliecog) [inline index.html]: having a class and an id with the same name on the same element is not verbose

Comment (charliecog) [reply index.html]: Does it need a both? could it just be referenced as ```<main>``` in the css as its the only one?

File: js/loadTrack.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
+$('.trackLink').click(function(e) {
+    var link = e.target.dataset.track
+        
+    if  (link === '1' && $('#track1').length === 0) {
```
Comment (charliecog) [inline js/loadTrack.js]: This large if statement is essentially repeated below. 
- abstract out into a function that can be called to save repeating yourselves

File: js/loadTrack.js [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+        $('#container').append($('<div>').load('includes/trackLayout1.html'))
+    } else if (link === '1' && $('#track1').length !== 0) {
```
Comment (charliecog) [inline js/loadTrack.js]: else if but no else at the end?

File: js/loadTrack.js [hunk 6]
```diff
@@ -0,0 +1,24 @@
+$('.trackLink').click(function(e) {
+    var link = e.target.dataset.track
+        
+    if  (link === '1' && $('#track1').length === 0) {
+        $('#container').append($('<div>').load('includes/trackLayout1.html'))
```
Comment (charliecog) [inline js/loadTrack.js]: To get rid of extra div assign the essential class and id stuff to this div (do this dynamically depending on track1/track2) and delete div from inside includes file
eg:
```$('#container').append($('<div id="track1" class="track">').load('includes/trackLayout1.html'))```

**Review by charliecog — CHANGES_REQUESTED** [review]
couple of bits to pick up, slight tweak to approach inside loadTrack.js

File: css/splash.css (hunk 1 above)
Comment (JamesCullen2) [reply css/splash.css]: done

File: includes/trackLayout1.html (hunk 2 above)
Comment (JamesCullen2) [reply includes/trackLayout1.html]: removed

File: index.html (hunk 3 above)
Comment (JamesCullen2) [reply index.html]: we omitted the class element, retained the id and changed the css styles files accordingly

File: js/loadTrack.js (hunk 4 above)
Comment (JamesCullen2) [reply js/loadTrack.js]: done

File: js/loadTrack.js (hunk 5 above)
Comment (JamesCullen2) [reply js/loadTrack.js]: omitted else if and replaced it with else and abstracted into function

File: js/loadTrack.js (hunk 6 above)
Comment (JamesCullen2) [reply js/loadTrack.js]: done

File: css/styles.css [hunk 7]
```diff
@@ -6,10 +6,14 @@ body {
     font-family: 'Press Start 2P', cursive;
 }
 
-.container {
+#container {
```
Comment (charliecog) [inline css/styles.css]: Id should be for hooking on JS not CSS, use ```<main>``` as discussed

File: css/styles.css [hunk 8]
```diff
@@ -6,10 +6,14 @@ body {
...
     width: 1000px;
     height: 800px;
     margin: 0 auto;
     border: 1px solid black;
 
+}
+
+.ie .splash {
+    background-color: blue;
```
Comment (charliecog) [inline css/styles.css]: should be a hex code (that matches the blue from the gradient version)

File: js/loadTrack.js [hunk 9]
```diff
@@ -1,24 +1,22 @@
 $('.trackLink').click(function(e) {
-    var link = e.target.dataset.track
-        
-    if  (link === '1' && $('#track1').length === 0) {
-        $('#container').append($('<div>').load('includes/trackLayout1.html'))
-    } else if (link === '1' && $('#track1').length !== 0) {
-        $('.track').css('display', 'none')
-        $('#track1').css('display', 'block')
-    }
+    var trackNo = e.target.getAttribute("data-track"),
```
Comment (charliecog) [inline js/loadTrack.js]: trackNum makes more sense as a variable name

Comment (charliecog) [reply js/loadTrack.js]: also single quotes for 'data-track'

File: js/loadTrack.js [hunk 10]
```diff
@@ -1,24 +1,22 @@
...
+
+    navigateLevels(trackElement, trackNo)
+})
 
-    if  (link === '2' && $('#track2').length === 0) {
-        $('#container').append($('<div>').load('includes/trackLayout2.html'))  
-    } else if (link === '2' && $('#track2').length !== 0) {
+function navigateLevels (trackElement, trackNo) {
+    if  (trackElement.length === 0) {
+        $('#container').append($('<div id="track' + trackNo + '" class="track">').load('includes/trackLayout' + trackNo + '.html'))
```
Comment (charliecog) [inline js/loadTrack.js]: single quotes for class="track"

File: css/styles.css [hunk 11]
```diff
@@ -6,10 +6,14 @@ body {
...
 
-.container {
+#container {
     width: 1000px;
     height: 800px;
     margin: 0 auto;
     border: 1px solid black;
 
+}
+
```
Comment (charliecog) [inline css/styles.css]: also add a display: block; to the main within ie9 to fix the border and positioning issues.

File: js/loadTrack.js [hunk 12]
```diff
@@ -1,24 +1,22 @@
...
-        $('#track2').css('display', 'block')
+        trackElement.css('display', 'block')
     }
 
-    $('#splash').css('display', 'none')  
-})
+    $('#splash').css('display', 'none')
+}
 
 $('body').on('click', '.trackReturn', function() {
```
Comment (charliecog) [inline js/loadTrack.js]: move to the top of the file with the other event listener for easy reading

**Review by charliecog — CHANGES_REQUESTED** [review]
changes to make, almost there!

File: css/styles.css (hunk 7 above)
Comment (nsantiagoblair) [reply css/styles.css]: 🎉

File: css/styles.css (hunk 8 above)
Comment (nsantiagoblair) [reply css/styles.css]: 🎉

File: js/loadTrack.js (hunk 9 above)
Comment (nsantiagoblair) [reply js/loadTrack.js]: 🎉

File: css/styles.css (hunk 11 above)
Comment (nsantiagoblair) [reply css/styles.css]: 🎉

File: js/loadTrack.js (hunk 12 above)
Comment (nsantiagoblair) [reply js/loadTrack.js]: 🎉

File: js/loadTrack.js (hunk 10 above)
Comment (nsantiagoblair) [reply js/loadTrack.js]: spoke about this one

**Review by charliecog — APPROVED** [review]

**Pull request merged**